tts-script-generation.md

```diff
@@ -0,0 +1,77 @@
+<img src="http://developer.download.nvidia.com/notebooks/dlsw-notebooks/riva_asr_deploy-eks/nvidia_logo.png" style="width: 90px; float: right;">
+
+
+# Data collection - Script Generation
+
+
+In this document, we will cover the outline to create a script for the voice actor to record. The script must cover all the phonemes with good coverage, must be relevant to the domain, and should have the necessary visual cues for the voice actor to get the desired cadence.
+
+
+## Identifying data sources:
+
+
+- The content that the voice actor will read needs to be aligned with the use case of the synthesized voice. If the voice is used for questions, have questions in the script. If the voice is going to be used for emotion, then have emotion in the script.
+
+
+- Another option is to synthetically generate text to be read, using a large language model like Megatron, BERT, or GPT-3. Since the data is coming from an LLM, it requires more manual normalization compared to other datasets.
```

Comment:
```suggestion
- Another option is to synthetically generate text to be read, using a large language model like Megatron, BERT, or GPT-3. Since the data is coming from an large language model (LLM), it requires more manual normalization compared to other datasets.
```